docs/guide/db-migrations.md

```diff
@@ -616,8 +619,8 @@ The migration command comes with a few command-line options that can be used to
 * `generatorTemplateFiles`: array (defaults to `[
         'create' => '@yii/views/createMigration.php',
         'drop' => '@yii/views/dropMigration.php',
-        'add' => '@yii/views/addMigration.php',
-        'remove' => '@yii/views/removeMigration.php',
+        'add_column' => '@yii/views/addMigration.php',
```

Comment:
It happens because the real file generation takes some time and expected `$class` name may be outdated to that moment.